core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -300,6 +300,7 @@ oppia.directive('conversationSkin', [
           $scope.hasFullyLoaded = false;
           $scope.recommendedExplorationSummaries = null;
           $scope.answerIsCorrect = false;
+          $scope.toPreviousState = false;
```

Comment:
Be more descriptive, e.g. $scope.pendingCardWasSeenBefore
  
(since this only applies when there's a pending card, right?)

Reply:
Yes, done